api/seccompprofile/v1beta1/seccompprofile_types.go

```diff
@@ -183,3 +212,30 @@ type SeccompProfileList struct {
 func init() { //nolint:gochecknoinits // required to init scheme
 	SchemeBuilder.Register(&SeccompProfile{}, &SeccompProfileList{})
 }
+
+func UnionSyscalls(baseSyscalls, appliedSyscalls []*Syscall) []*Syscall {
```

Comment:
Do we wanna make this private? This function should be only used internally since it does not handle arches and arguments.

Reply:
I agree, I don't think we need to make this public for now.

Reply:
I moved UnionSyscalls into an util module where it can be reused by both the deamon and the controller